scripts/plotting/global_latlon_map.py

```diff
@@ -44,6 +47,43 @@ def global_latlon_map(adf):
     # - Take difference, calculate statistics
     # - make plot
 
+
+    #
+    # Use ADF api to get all necessary information
+    #
+    var_list = adfobj.read_config_var("diag_var_list")
+    basic_info_dict = adfobj.read_config_var("diag_basic_info")
+    plot_location = basic_info_dict["cam_diag_plot_loc"]
+    compare_obs = basic_info_dict["compare_obs"]
+
+    case_info_dict = adfobj.read_config_var("diag_cam_climo")
+    case_name = case_info_dict["cam_case_name"]
+    model_rgrid_loc = case_info_dict["cam_regrid_loc"]
```

Comment:
In order to run, it needs to look like this:

    model_rgrid_loc = adfobj.get_basic_info("cam_regrid_loc", required=True)